hero/full-client/index.cjs

```diff
@@ -1,3 +0,0 @@
-require('./index.js');
```

Comment:
Why delete the commonjs support? Are we gonna get rid of default exports?

Reply:
It's not deleted. That might have been related to the rename from full-client to main. I reverted it back to full-client.

Reply:
Oh, I meant to delete this comment after I saw main